src/Controller/UserController.php

```diff
@@ -234,20 +224,29 @@ public function editUser(
 
         try {
             $userEdit = $this->transformer->editTransform($userDTO, $user);
-        } catch (EntityNotFound $exception) {
-            return new JsonResponse(
-                [
-                    'code' => Response::HTTP_NOT_FOUND,
+        } catch (TransformerException $exception) {
+            if ($exception instanceof EntityNotFound) {
```

Comment:
If you handle the exceptions differently, use multiple `catch{}` blocks and each can have different exceptions to catch.